shared/server-commands/videos/live-command.ts

```diff
@@ -118,10 +118,11 @@ export class LiveCommand extends AbstractCommand {
 
   async quickCreate (options: OverrideCommandOptions & {
     saveReplay: boolean
+    replaySettings?: { privacy: VideoPrivacy }
```

Comment:
I think you can remove this argument, and automatically set it to private since it's a "quick" create